packages/account/src/Sections/Security/ApiToken/api-token.tsx

```diff
@@ -1,4 +1,263 @@
+                                                                value={values.token_name}
+                                                                onChange={e => {
+                                                                    setFieldTouched('token_name', true);
+                                                                    handleChange(e);
+                                                                }}
+                                                                onBlur={handleBlur}
+                                                                hint={localize(
+                                                                    'Length of token name must be between 2 and 32 characters.'
+                                                                )}
+                                                                required
+                                                                error={
+                                                                    touched.token_name && errors.token_name
+                                                                        ? errors.token_name
+                                                                        : undefined
+                                                                }
+                                                            />
+                                                        )}
+                                                    </Field>
+                                                    <Button
+                                                        className={classNames(
+                                                            'dc-btn__button-group',
+                                                            'da-api-token__button',
+                                                            {
+                                                                'da-api-token__button--success': isSuccess,
+                                                            }
+                                                        )}
+                                                        type='submit'
+                                                        is_disabled={
+                                                            !dirty ||
+                                                            isSubmitting ||
+                                                            !isValid ||
+                                                            !selectedTokenScope(values).length
+                                                        }
+                                                        has_effect
+                                                        is_loading={isSubmitting}
+                                                        is_submit_success={isLoading}
+                                                        text={localize('Create')}
```

Comment:
can we pass this text as children to the Button component so that we can use Localize

Reply:
Will make this change. Also its there in other APIToken sub components